### ISSUE TITLE
Expose generator factory to use as library

### ISSUE THREAD

**Issue by marcoqu** [xiag-ag/typescript-to-json-schema#31]
Could you add the factory function `createGenerator` to the types you export?
I'm working on a vscode integration of this tool, and this would help in creating a `SchemaGenerator` from code.

**Comment by mrix** [comment]
Hi @marcoqu. Please take a look at #33. Here's an example how to use it with the new types:
```ts
// test.ts

// the both imports are valid
import { Factory } from "typescript-to-json-schema";
import { createGenerator } from "typescript-to-json-schema/types/Factory";

Factory.createGenerator({
    type: "SomeType",
    path: "./tsconfig.json",

    expose: "export",
    jsDoc: "extended",
    sortProps: true,
    topRef: true,
});
createGenerator({
    type: "SomeType",
    path: "./tsconfig.json",
    
    expose: "export",
    jsDoc: "extended",
    sortProps: true,
    topRef: true,
});
```

```json
// tsconfig.json
{
  "compilerOptions": {
    "baseUrl": ".",
    "paths": {
      "typescript-to-json-schema": ["node_modules/typescript-to-json-schema/index"],
      "typescript-to-json-schema/types/*": ["node_modules/typescript-to-json-schema/types/*"]
    }
  },
  "include": [
    "test.ts"
  ]
}
```

How about this?

**Comment by marcoqu** [comment]
Great, thanks!

**Comment by mrix** [comment]
Available in v0.6.0
